### DISCUSSION TITLE
ColorManager v2.5

### DISCUSSION THREAD

**Discussion by aljawaid** [aljawaid/ColorManager#24] [Announcements]
### What's Changed
_(most recent changes are listed on top):_
- f2a5884 Better plugin description
- e9722dc Update mention of total count in description
- 20b6fce NEW: Add 5 New Colours to Extended Palette
- 040c193 Add `magenta` to Extra Colour Palette- #3
- 41f6262 Add `black_on_yellow` to Extra Colour Palette- #3
- 74027a6 Add `black_on_grey` to Extra Colour Palette- #3
- 6491b7b Add `grey_on_black` to Extra Colour Palette- #3
- 733a09f Add `bright_white` to Extra Colour Palette - Requested by @JustFxDev
- 2f64cf8 NEW: Add Colour Name Translations #22
- 3bb4960 FIX: Translations Coding Consistency #21
- 9840783 FIX: Conflicting Asset Path Between Kanboard Versions
- dbd1653 Translate Colour Name in Tooltip
- fe6e631 NEW: Add `de_DE` Translations #21 -- thanks @JustFxDev
- 903a069 FIX: Remove Unused String
- 381fcfd FIX: `white_on_green` too dark
- 0d12f1f FIX: `bright_yellow` not bright enough
- ff7125a Fixed path to svg icons

<hr /><em>This discussion was created from the release <a href='https://github.com/aljawaid/ColorManager/releases/tag/v2.5'>ColorManager v2.5</a>.</em>

**Comment by aljawaid** [comment]
@JustFxDev can you translate the colours for `de_DE` please?

**Reply by aljawaid** [reply]
0e62000

**Comment by JustFxDev** [comment]
Done. See Zip.

I pulled it, but did not see the changes. Maybe i do not know how to get a uncommited change. I attached the files here. Translation works. I can see them in the drop down when creating or editing a task 👍 

Cheers, Fx
[de_DE.zip](https://github.com/aljawaid/ColorManager/files/10327403/de_DE.zip)

![grafik](https://user-images.githubusercontent.com/82320201/210138133-f99f4358-b844-4e98-922d-7272e8091da4.png)

**Reply by aljawaid** [reply]
thanks, but also you need to keep your file clean of whitespaces. I fixed it here https://github.com/aljawaid/ColorManager/commit/81a2995b92b0ff9f1b4331b6d72e1bccc16e70fd

**Comment by JustFxDev** [comment]
Thanks for that advice. I doublecheck my translation sheet

**Comment by JustFxDev** [comment]
You mean the double space before and after `  =>  `, right?

**Reply by aljawaid** [reply]
yes
